includes/discovery/sensors/current/mgeups.inc.php

```diff
@@ -3,7 +3,7 @@
 // MGE UPS
 if ($device['os'] == 'mgeups') {
     echo 'MGE ';
-    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.1', '-OsqnU'));
+    $oids = trim(snmp_walk($device, '1.3.6.1.4.1.705.1.7.2.1.5', '-OsqnU'));
```

Comment:
And to clarify, the OIDs that I used to count the number of phases are used about 4 lines down to gather the data to add to the database.